consistent weapon special text + minor fixes

nemesis-vampy opened the pull request:
- all non-dual wield weapon specials print text, styled consistently
- use correct (i think?) name for the quick launcher
- fix medkit low hp message not working
- fix upgraded shotgun ejecting the wrong shells when swapping types
- fix pb_hazardmovement cvar not affecting dashes

nemesis-vampy committed:
fix medkit low hp message

nemesis-vampy committed:
fix quick launcher name

nemesis-vampy committed:
shotgun special ejects correct shells with upgrade

nemesis-vampy committed:
consistent weapon special text

nemesis-vampy committed:
pb_hazardmovement affects dashing again

nemesis-vampy committed:
fix mugshot scaling

nemesis-vampy committed:
faster bob correction when landing

looks a bit awkward but should be better than it was before

popguy12 commented:
>pb_hazardmovement affects dashing again
was this broken???????

popguy12 commented:
everything else looks fine, i guess i never noticed that was broken
only thing that looks off is the mugshot scaling change, miiight need to look into checking if HUD Preserves Aspect Ratio is on or off for that to make sure it doesnt look super skinny

nemesis-vampy commented:
as far as i can tell pb's hud just completely ignores that cvar

nemesis-vampy committed:
clean up weapon pickup messages + tags

nemesis-vampy committed:
fix offsets on revolver ads fire

nemesis-vampy committed:
axe + keys pickup billboarding

these are the only pickups without standard billboarding, so i'm assuming that's the intended style

nemesis-vampy committed:
fix pb_keepweapons cvar

nemesis-vampy committed:
more fist select sounds

feels weird without it imo

nemesis-vampy committed:
fix gearbox cursor

translation completely fucks it, better to just recolor it

popguy12 commented:
it does, however its on by default in the engine, so the mugshot ends up looking like this
<img width="683" height="227" alt="image" src="https://github.com/user-attachments/assets/f2b8788d-4f88-44d3-a704-8115e6f3aa3b" />
id check with Carrot though, cause weve kinda been moving towards forcefully correcting rendering of things to actually be square pixels, mostly on the weapons, but somewhat on the HUD too as long as that option is turned off

nemesis-vampy committed:
only stretch mugshot if hud isn't already stretched

nemesis-vampy committed:
..and only if custom mugshot is disabled

since other mods' mugshots would have been intended to render withvanilla's hud scaling

nemesis-vampy commented:
should all be fixed now
